.claude/skills/supabase-jwt-auth/references/troubleshooting.md

```
@@ -0,0 +1,261 @@
+  const payload = JSON.parse(Buffer.from(accessToken.split('.')[1], 'base64').toString());
+
+  return NextResponse.json({
+    user_app_metadata: session.user.app_metadata, // カスタムクレームなし
+    jwt_payload: payload, // カスタムクレームあり
+  });
+}
+```
+
+**Hookが正常動作しているか確認:**
+PostgreSQL Logsで以下のメッセージを確認：
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)